tree-sitter/src/TreeSitter/Range.hs

```diff
@@ -0,0 +1,7 @@
+module TreeSitter.Range
+( Range(..)
+) where
+
+  -- | A half-open interval of integers, defined by start & end indices.
+  data Range = Range { start :: {-# UNPACK #-} !Int, end :: {-# UNPACK #-} !Int }
+    deriving (Eq, Show)
```

Comment:
Looks like this is indented a little strangely.

Reply:
copy 🍝mishaps.